tools/aws-ip.sh

```diff
@@ -1,3 +1,8 @@
 ipv4=$(curl http://169.254.169.254/latest/meta-data/public-ipv4)
 
-echo '\n\nAPI_URL=http://'$ipv4':3000' >> ../telescope/.env
+sed -r \
```

Comment:
Let's generalize this script so we can use it both here and in the GitPod init (e.g. change the name, split out the aws vs. gitpod stuff).  This can happen in a follow-up.

cc @DukeManh.

Reply:
Let's export the URLs to the environment variable so we can reuse the same script without passing parameters. Either in this PR or a follow-up